### PR TITLE
adapt to changes proposed in #192

### PR DESCRIPTION
**Pull request by rajatchopra**
Signed-off-by: Rajat Chopra <rchopra@redhat.com>

Depends on #192 
@dcbw just a basic update. PTAL.

### CODE REVIEW

**Comment by shettyg** [comment]
@rajatchopra Can you please change the commit subject to be independent of a different commit.

File: go-controller/README.md [hunk 1]
```diff
@@ -12,10 +12,12 @@ make clean
 make
 ```
 
-Then find the executable here : go-controller/_output/go/bin/ovnkube
+Then find the executables here : go-controller/_output/go/bin/
 
 'ovnkube' is the single all-in-one executable useful for deploying the ovn cni plugin for a kubernetes deployment
 
+'ovn-k8s-cni-overlay' is the cni executable to be planted under /opt/cni/bin so that it can be invoked for each pod event
```
Comment (pecameron) [inline go-controller/README.md]: This is moved to /usr/libexec

File: go-controller/README.md [hunk 2]
```diff
@@ -73,6 +75,14 @@ With the above the master ovnkube controller will initialize the central master
 
 #### Initialize a newly added node for the OVN network
 
+Remember to install the cni binary first. Use the Makefile, or, manually copy 'ovn-k8s-cni-overlay' to /opt/cni/bin (or the appropriate directory that kubernetes will use to look for the plugin).
```
Comment (pecameron) [inline go-controller/README.md]: /opt/cni/bin is changed to /usr/libexec

File: go-controller/README.md (hunk 1 above)
Comment (dcbw) [reply go-controller/README.md]: Only on Fedora or other distros that don't like /opt though.  It'll still be /opt/cni/bin for others.

File: go-controller/README.md (hunk 2 above)
Comment (dcbw) [reply go-controller/README.md]: Same as above...

File: go-controller/README.md (hunk 1 above)
Comment (pecameron) [reply go-controller/README.md]: Question: how do I build it for the fedora RPM?

**Pull request closed without merging**